server/routes/users.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const {
+  join,
+  login,
+  userInformation,
+  infoToUpdate,
+  updateUserInformation,
+} = require("../controller/usersControlloer");
+
+router.use(express.json());
+
+router.post("/join", join); // 회원가입
+router.post("/login", login); // 로그인
+router.get("", userInformation); // 메인페이지 location, nickname 조회
```

Comment:
/users 경로를 표현하려고 하신거라면 아래와 같이 작성해주세요~

```suggestion
router.get("/", userInformation); // 메인페이지 location, nickname 조회
```